.github/workflows/react_tests.yml

```diff
@@ -0,0 +1,50 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: React Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+defaults:
+  run:
+    working-directory: katello
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        path: katello
+    - name: Use Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.x'
+    - uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 2.6
+    # We could update the postinstall action for foreman to look for an environment variable for plugin webpack dirs
+    # before kicking off the ruby script to find them, setting that would eliminate the ruby dep and running `npm i` in Katello.
+    - name: Install bundler (needed to find Foreman plugins)
```

Comment:
I thought I remembered reading somewhere bundler is preinstalled on Github action runners?